contracts/Penalizer.sol

```diff
@@ -26,8 +26,30 @@ contract Penalizer is IPenalizer{
 
     }
 
-    modifier relayManagerOnly(IRelayHub hub) {
-        require(hub.isRelayManagerStaked(msg.sender), "Unknown relay manager");
+    mapping( bytes32=>bool ) public commits;
```

Comment:
Aslo, please keep the code style at least somewhat consistent. No spaces inside braces, spaces after commas etc.